.github/workflows/release_check_version.yaml

```diff
@@ -33,7 +33,7 @@ jobs:
           GITHUB_REF_VERSION: ${{ github.ref_name }}
         run: |
           # Extract resonate version
-          RESONATE_VERSION=$(./resonate -v | awk '{print $3}')
+          RESONATE_VERSION=$(./resonate -v | awk '{print $3}' | sed 's/^v//g')
```

Comment:
This will solve the workflow, if we move forward with this change. But, at the moment, I'm not aware of what other potential breaking change this might introduce as I still don't have the full picture of the resonate landscape.